roaring/roaring_test.go

```diff
@@ -1137,3 +1112,113 @@ func TestBitmap_Intersect(t *testing.T) {
 		t.Fatalf("Counts do not match %d %d", bm0.Count(), result.Count())
 	}
 }
+
+func BenchmarkBitmap_IntersectionCount_ArrayBitmap(b *testing.B) {
+	data := &benchmarkBitmapIntersectionCountData
+	if data.a == nil {
+		const max = (1 << 24) / 64
+
+		// Build bitmap with array container.
+		data.a = roaring.NewBitmap()
+		for i, n := 0, rand.Intn(roaring.ArrayMaxSize); i < n; i++ {
```

Comment:
you can't compare this benchmark to others because it puts a random amount of data in the array. The results will be wildly different if the rand returns a 1 vs a 4000. I think an unseeded rand will actually return the same results every time, but then we might as well just define the data statically instead of relying on that.

Reply:
good call. I'm change to 2/3 array max, just because i want more than half